src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
```

Comment:
Be consistent with your margins. Add only top or bottom, but not both to avoid potential margin collapse. In this case, you have a margin-bottom, which could be alright if you're following a top-to-bottom margin policy, but ensure this is consistent across your styles.